syntax = "proto3";
import "frame_config.proto";
import "label_config.proto";


message PredictionModelConfig {
    string name = 1;
    FrameConfig frame_config = 3;
    LabelConfig label_config = 4;
}
